src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
```

Comment:
Do we generate documentation from the DAO layer?

I thought, we only need the API documentation annotations in the REST layer.